src/index.spec.ts

```diff
@@ -528,6 +524,211 @@ describe('initialization options', () => {
     expect(callCount).toBe(0);
   });
 
+  it('Uses its cache', async () => {
+    // Mock fetch so first call works, second fails
+    let fetchCallCount = 0;
+    let fetchResolveCount = 0;
+    const fetchResolveDelayMs = 500;
+    global.fetch = jest.fn(() => {
```

Comment:
It might be better to use spyFn = jest.spyOn(global, 'fetch') in this case so that you can restore the original function via spy.mockRestore() in an afterAll

Reply:
great call!

Reply:
Alright we need to do this after all because jest doesn't allow `fetch` in global by default; probably because node didn't have it until version 18 🤷‍♂️ 

https://github.com/jestjs/jest/issues/2071

Doing it the "correct" way results in an error 😞

Reply:
Ah ok